resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -267,37 +260,19 @@ The `tidyverse` R package (and some others) have external dependencies on Ubuntu
 sudo apt install libcurl4-openssl-dev libssl-dev libxml2-dev
 ```
 
-We will install the key R packages needed for the start of our program now:
+Next, install the key R packages needed for the start of MDS program,
+by typing the following into the R terminal in RStudio:
 
 ```
-install.packages(c('tidyverse', 'tinytex', 'blogdown', 'xaringan', 'renv'))
+install.packages(c('tidyverse', 'tinytex', 'blogdown', 'xaringan', 'renv', 'IRkernel'))
```

Comment:
Let's remove `tinytex` here and just install it later under the LaTeX section, as I think this logically makes more sense (as they need another command to install it as well).
```suggestion
install.packages(c('tidyverse', 'blogdown', 'xaringan', 'renv', 'IRkernel'))
```

Reply:
Yeah maybe... I think conceptually it makes sense so folks know what the individual things are for (tinyTex for LaTeX, IRkernel for the R kernel for Jupyter, and the rest are R packages for other stuff).